doc/src/sgml/xlstproc_depfile_wrapper.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+import subprocess
+import sys
+import argparse
+from unittest import result
+
+parser = argparse.ArgumentParser(
+    description='generate dependency file for docs')
+
+parser.add_argument('--basename', type=str, required=False, nargs='+')
```

Comment:
What does 'basename' mean here?

Reply:
It is set to '@OUTPUT@' and used for getting targets' file names in depfile. Perhaps 'targetname' can be used instead of basename?